backend/protocol_rpc/rpc_methods.py

```
@@ -0,0 +1,525 @@
+        msg_handler=msg_handler,
+        transactions_parser=transactions_parser,
+        consensus_service=consensus_service,
+        signed_rollup_transaction=signed_rollup_transaction,
+        sim_config=sim_config,
+    )
+
+
+@rpc.method("eth_getTransactionCount")
+def eth_get_transaction_count(
+    address: str,
+    transactions_processor: TransactionsProcessor = Depends(get_transactions_processor),
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)